pkg/component/exec_handler.go

```diff
@@ -55,7 +55,7 @@ func (o *execHandler) ApplyOpenShift(openshift v1alpha2.Component) error {
 func (o *execHandler) Execute(command v1alpha2.Command) error {
 	msg := o.msg
 	if msg == "" {
-		msg = fmt.Sprintf("Executing %s command %q on container %q", command.Id, command.Exec.CommandLine, command.Exec.Component)
+		msg = fmt.Sprintf("Executing %s command on container %q", command.Id, command.Exec.Component)
```

Comment:
@rm3l just to understand better, this function is used to execute command inside the container?

Reply:
Yes, it is an implementation of a generic [`libdevfile.Handler` interface](https://github.com/redhat-developer/odo/blob/main/pkg/libdevfile/libdevfile.go#L26), used to "handle" commands defined in the Devfile that odo needs to take into account, depending on their type (`apply`, `exec`, ...).
Here it is the handler for `exec` commands, and it delegates to an `exec` client that does the job of executing the command inside the container (via again a generic platform client interface, with different implementations for Podman and K8s).